src/MspFcComms.cpp

```diff
@@ -78,12 +78,14 @@ namespace FcComms
     {
         uint16_t raw_values[18];
         getRawRC(raw_values);
-        char RC_info[100];
-        int j = 0;
-        for (int i = 0 ; i < 18 ; i++) {
-            j+=sprintf(&RC_info[j], ", %d", raw_values[i]);
-        } //This is Levi's for loop
-        ROS_INFO(RC_info);
+
+        std::string RC_info;
```

Comment:
Yeah 100 bytes wasn't long enough anymore and the stack was being smashed. I wrote it originally so my bad.